pygsp2/tests/test_utils2.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+"""
+Test suite for the utils module of the pygsp2 package.
+
+"""
+
+import unittest
+import pandas as pd
+from unidecode import unidecode
+
+from pygsp2 import utils_examples
+
+
+class TestCase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def test_graphs(self):
+        edgesfile='../data/santiago_metro_stations_connections.txt',
+        coordsfile='../data/santiago_metro_stations_coords.geojson'
```

Comment:
Note that this files should not be in the repo.

Reply:
The original function in "utils_examples.py" was hard coded/ bound to data contained in the examples folder, which was not accessible from within the function, easiest way to solve it was to move said files to the data folder of PyGSP.

Reply:
https://github.com/gsp-eeg/pygsp2/pull/25#discussion_r1657621789